Sources/Rendering/Core/InteractorStyle/index.js

```diff
@@ -194,8 +194,6 @@ function vtkInteractorStyle(publicAPI, model) {
   publicAPI.startState = (state) => {
     model.state = state;
     if (model.animationState === States.IS_ANIM_OFF) {
-      // const rwi = model.interactor;
-      // rwi.getRenderWindow().setDesiredUpdateRate(rwi.getDesiredUpdateRate());
       publicAPI.invokeStartInteractionEvent({ type: 'StartInteractionEvent' });
```

Comment:
@martinken we could also have that StartInteractionEvent/EndInterationEvent be static object defined at the top of the module.